raptorWeb/authprofiles/models.py

```diff
@@ -84,13 +57,58 @@ class UserProfileInfo(models.Model):
         max_length=80, 
         blank=True
     )
-
     def __str__(self):
-        try:
-            return self.user.username
-        except:
-            return self.discord_info.tag
+        return f'UserProfileInfo#{self.id}'
 
     class Meta:
         verbose_name = "User - Extra Information"
         verbose_name_plural = "Users - Extra Information"
+
+class RaptorUser(User):
```

Comment:
Possible to have RaptorUser inherit from `AbstractBaseUser` instead? This might force us to re-implement things that Django has done for us, and the return from doing such may be negligible. Will do research!

Reply:
Inheriting from AbstractUser suits our needs